src/hooks/useDeppLinkTransferParams.ts

```diff
@@ -0,0 +1,27 @@
+import { ChainId, coalesceChainId, ChainName } from "@certusone/wormhole-sdk";
+import { useMemo } from "react";
+
+function parseChain(chain: string | null = null): ChainId {
+    if (chain) {
+        const chainId = parseInt(chain);
+        try {
+            if (isNaN(chainId)) {
+                return coalesceChainId(chain as ChainName);
+            } else {
+                return coalesceChainId(chainId as ChainId);
+            }
+        } catch (err) {
+            console.error("Invalid path params specified.", err);
+            return NaN as ChainId;
+        }
+    } else {
+        return NaN as ChainId;
+    }
+}
+
+export function useDeppLinkTransferParams(search: string) {
```

Comment:
```suggestion
export function useDeepLinkTransferParams(search: string) {
```

Deep link or Depp link?

Reply:
good call